app/jobs/decidim/action_delegator/twilio_send_sms_job.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'twilio-ruby'
+
+module Decidim
+  module ActionDelegator
+    class TwilioSendSmsJob < ApplicationJob
+      queue_as :default
+
+      def perform(sender_name, mobile_phone_number, message)
+        @sender_name = sender_name
+        @mobile_phone_number = mobile_phone_number
+        @message = message
+
+        send_sms!
+      end
+
+      private
+
+      attr_reader :sender_name, :mobile_phone_number, :message
+
+      def send_sms!
+        client.messages.create(
+          from: sender_name,
```

Comment:
According to https://www.twilio.com/docs/sms/quickstart/ruby?code-sample=code-send-sms-using-twilio-1&code-language=Ruby&code-sdk-version=5.x, it's not a name but a phone number so I'd rename this var to `from_phone_number` or similar